pkg/newrelic/client.go

```diff
@@ -280,9 +498,23 @@ func (c *Client) Query(ctx context.Context, query string, variables map[string]i
 
 	vars["userId"] = c.apikey
 
+	return c.doRequest(ctx, query, vars, res)
+}
+
+func (c *Client) Mutate(ctx context.Context, query string, variables map[string]interface{}, res interface{}) error {
+	vars := make(map[string]interface{}, len(variables)+1)
+
+	for k, v := range variables {
+		vars[k] = v
+	}
```

Comment:
Just curious, but why does Query need an API key in the userId var but Mutate doesn't?

Reply:
This is my bad of not understanding graphql properly 😄 When I was testing this API, I thought `requestContext` is mandatory to make a request, but I can see that the problem was elsewhere. I'll remove unnecessary code.

Reply:
@ggreer I removed `Query` and `Mutate` methods. It's using common `doRequest` method now.